mosaic-terminal/build.gradle

```diff
@@ -14,7 +14,7 @@ kotlin {
 		compilations.main.compileTaskProvider.configure {
 			doFirst {
 				def files = project.fileTree('src/jvmMain/resources').files
-				if (files.size() != 6) {
+				if (files.size() != 7) {
```

Comment:
There was an error, as 7 files were generated, probably after adding `risc` to build native artifacts.

Reply:
Hmm weird. Why didn't that PR fail?

Reply:
I guess it was cached because the inputs didn't change. I need to hook into the process resources task or whatever instead.

Reply:
#476